tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -52,6 +65,60 @@ typedef struct _test_globals
 static test_globals_t _globals;
 static volatile bool _globals_initialized = false;
 
+static void
+_impersonate_user()
+{
+    printf("Impersonating user [%s].\n", _user_name.c_str());
+    bool result = ImpersonateLoggedOnUser(_globals.user_token);
+    REQUIRE(result == true);
+}
+
+static void
+_revert_to_self()
+{
+    printf("Reverting to self.\n");
+    RevertToSelf();
+}
+
+typedef class _impersonation_helper
+{
+  public:
+    _impersonation_helper(execution_mode_t mode)
+    {
+        if (mode == Standard) {
```

Comment:
```suggestion
        if (mode == execution_mode_t::Standard) {
```
be consistent with line 138 etc.   Either use a C enum and add a type-specific prefix like EXECUTION_MODE_STANDARD_USER, or else use C++ style like in suggestion above, as line 138 already does.  Look for all occurrences in this file.

Reply:
Fixed. Changed all instances to C++ style.